website/docs/containers/registries/index.md

```diff
@@ -2,18 +2,275 @@
 sidebar_position: 3
 title: Registries
 description: Working with container registries
-tags: [podman-desktop, containers, registries]
+tags: [podman-desktop, containers]
 keywords: [podman desktop, podman, containers, registries]
 hide_table_of_contents: false
+image: /img/docs/containers/registries/img/registries.png
 ---
 
-# Working with container registries
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-Podman Desktop has access to public registries by default.
+# Setting up container registries
 
-You might want to configure Podman Desktop to:
+## Overview
 
-- [Authenticate to a preconfigured registry](/docs/containers/registries/authenticating-to-a-preconfigured-registry).
-- [Add an Insecure Registry to Podman Desktop](/docs/containers/registries/insecure-registry).
+Podman Desktop allows you to configure Podman container registries from its graphical user interface: no need to use the command line interface.
 
-![Registries page](img/registries.png)
+If you are a developer who deals with containers, often finds yourself pulling and pushing container images – this tutorial is tailor-made for you!
+
+By the end of this tutorial, you will be able to [set up a container registry in the most straightforward context: a pre-configured container registry](#setting-up-a-pre-configured-container-registry), and also in more complex contexts such as [a custom container registry](#setting-up-a-custom-container-registry), or [a custom container registry with an insecure certificate, such as a self-signed certificate](#setting-up-a-custom-container-registry-with-an-insecure-certificate).
```

Comment:
This is not easy to read and to scan.
As a developer (and reader) and want to quickly identify the content of the doc and using a bullet list would be so much more efficient.

Reply:
Of course, it is not easy to read and scan. We had a bullet list and you requested it to be replaced by a sentence using the exact same argument.